sonar-perl-plugin/src/test/java/com/github/sonarperl/critic/PerlCriticIssuesLoaderSensorTest.java

```diff
@@ -56,7 +57,7 @@ public void should_not_throw_on_report_file_not_found() {
         String relativePath = "lib/Sample/Project.pm";
         inputFile(relativePath);
         context.settings().setProperty(PerlCriticProperties.PERLCRITIC_REPORT_PATH_KEY, "src/test/resources/basic/perlcritic_nonexistant_report.txt");
```

Comment:
Complete class `Settings` is deprecated and will be removed with Sonarqube 8.

Reply:
Thanks for the reminder.